setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name="ossfs",
-    version="0.1",
+    version="20210609",
```

Comment:
What happened to the version format? Seems like the dots are missing

Reply:
I also see that this was released on pypi already, you might have to remove that in `https://pypi.org/manage/project/ossfs/releases/`

Reply:
@efiop , Yes, the version was forgotten to change last time. So what should I do next? First submit a release without version then change to the new format?

Reply:
@karajan1001 Just delete the `20210609` version from pypi through that link above and then publish a correct one.